### PR TITLE
separating notification channels -paging and non-paging

### PR DESCRIPTION
**Pull request by yuriatgoogle**
Fixes #

## Proposed Changes

* Added paging and non-paging notification channels
* Specified paging alerts for SLO and latency violations
* Specified paging and non-paging alerts 

**Release Note**

```release-note
Creates separate paging and non-paging alert mechanisms.
```

### CODE REVIEW

File: terraform/alerting/notifications.tf [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
   labels       = each.value.labels
   depends_on = [
     null_resource.manual-step-to-enable-workspace
   ]
 
-  for_each = var.alert-notification-channels
+  for_each = var.alert-notification-channel-paging
+}
+
+resource "google_monitoring_notification_channel" "nonpaging" {
```
Comment (sethvargo) [inline terraform/alerting/notifications.tf:14]: Nit: It's standard to use dashes for separators in resource names (e.g. this should be `"non-paging"`)

Comment (yuriatgoogle) [reply terraform/alerting/notifications.tf]: Addressed, thank you.

File: terraform/alerting/notifications.tf [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+  provider     = google-beta
+  project      = var.project
+  display_name = "Non-paging Notification Channel"
+  type         = each.key
+  labels       = each.value.labels
+  depends_on = [
+    null_resource.manual-step-to-enable-workspace
+  ]
+
+  for_each = var.alert-notification-channel-nonpaging
```
Comment (mariliamelo) [inline terraform/alerting/notifications.tf:24]: should this be `non-paging` as well?

File: terraform/alerting/variables.tf [hunk 3]
```diff
@@ -40,12 +40,30 @@ variable "alert-notification-channels" {
...
+        channel_name = "#paging-channel"
+        auth_token   = "paging-token"
       }
     }
   }
-  description = "Notification channels"
+  description = "Paging notification channel"
+}
+
+variable "alert-notification-channel-nonpaging" {
```
Comment (mariliamelo) [inline terraform/alerting/variables.tf:51]: `non-paging` ?

**Review by mariliamelo — COMMENTED** [review]
Let's use only `non-paging` then

File: terraform/alerting/notifications.tf (hunk 2 above)
Comment (yuriatgoogle) [reply terraform/alerting/notifications.tf]: Yes, thank you. Done.

File: terraform/alerting/variables.tf (hunk 3 above)
Comment (yuriatgoogle) [reply terraform/alerting/variables.tf]: Done, thank you.

**Pull request merged**